### PR TITLE
Update about-the-dependency-graph.md

### LINKED ISSUE #17111: Inaccurate information on "About the dependency graph" page "Supported package ecosystems" section
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems

### What changes are you suggesting?

- NuGet is the package manager for .NET
- C++ is not a .NET language

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by rukasakurai**
- NuGet is the package manager for .NET
- C++ is not a .NET language

### Why:
Inaccurate information on "About the dependency graph" page "Supported package ecosystems" section

Closes https://github.com/github/docs/issues/17111

### What's being changed:
- Package manager for .NET to NuGet
- Removing C++ from the list of .NET languages

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@rukasakurai 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Sandrawillims — COMMENTED** [review]
`@${context.payload.sender.login} opened

**Review by Sandrawillims — COMMENTED** [review]
content/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph.md

**Review by SMAKSS — COMMENTED** [review]
I guess `NuGet` is only one of the options and we also do have [`.NET CLI`](https://docs.microsoft.com/en-us/dotnet/core/tools/) for managing all the `.NET` application tools.

**Comment by SIkebe** [comment]
Hi @rukasakurai , I raised a similar PR (#17110)  a few hours before this one 😄
I'm not very familiar with C++, but NuGet supports C++ according to some docs.

>**What programming languages are supported by NuGet?**
NuGet generally works for .NET languages and is designed to bring .NET libraries into a project. Because it also supports MSBuild and Visual Studio automation in some project types, it also supports other projects and languages to various degrees.
>
>The most recent version of NuGet supports C#, Visual Basic, F#, WiX, and **C++**.

https://docs.microsoft.com/en-us/nuget/resources/nuget-faq#what-programming-languages-are-supported-by-nuget-

NuGet for C++
https://devblogs.microsoft.com/cppblog/nuget-for-c/

**Comment by SIkebe** [comment]
> I guess `NuGet` is only one of the options and we also do have [`.NET CLI`](https://docs.microsoft.com/en-us/dotnet/core/tools/) for managing all the `.NET` application tools.

"NuGet" **is** the package manager. the NuGet client in Visual Studio, nuget.exe, and .NET CLI all implements [NuGet Server API](https://docs.microsoft.com/en-us/nuget/api/overview) to do operations related to NuGet.
So "NuGet" is suitable here for package manager name.

File: content/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The recommended formats explicitly define which versions are used for all direct
 | Package manager | Languages | Recommended formats | All supported formats |
 | --- | --- | --- | ---|
 | Composer             | PHP           | `composer.lock` | `composer.json`, `composer.lock` |
-| `dotnet` CLI | .NET languages (C#, C++, F#, VB)  |   `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj` |  `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj`, `packages.config` |
+| NuGet | .NET languages (C#, F#, VB)  |   `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj` |  `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj`, `packages.config` |
```
Comment (SIkebe) [inline content/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph.md:69]: Is there any reason why you removed C++ and kept `.vcxproj`?

Comment (SIkebe) [reply content/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph.md]: > but outside of the () for .NET Languages

Yeah, that should be fine👍 Thanks!

**Comment by felicitymay** [comment]
@rukasakurai and @SIkebe - many thanks for raising this problem with the docs ✨  and for your constructive discussion in this pull request 💖 

We're checking on the reasons for the original content and will get back to you as soon as we can.

**Review by felicitymay — APPROVED** [review]
Many thanks for confirming that this is a good change @annarosenthal 💖 

We'll merge this PR 🚀

**Pull request merged**